test/test_reason.re

```diff
@@ -1175,6 +1175,32 @@ let testSvg = () =>
     );
   });
 
+let testStrictModeArbitraryVal = ref("ja");
```

Comment:
For the test to run properly, please update the version of React to 18.

Reply:
Updated to v18.1.0!